aiida/orm/groups.py

```diff
@@ -382,3 +420,15 @@ def get_schema():
                 'type': 'unicode'
             }
         }
+
+
+class AutoGroup(Group):
+    """Group to be used to contain all nodes from an export archive that has been imported."""
```

Comment:
Made it slightly different, because it can also be activated outside of `verdi run`. Made it:

    """Group to be used to contain selected nodes generated while `aiida.orm.autogroup.CURRENT_AUTOGROUP` is set."""